# Statistics

## Extension

File Extension | Count
---------- | ----- 
`gff3` | 153
`gff` | 11

## Headers

Header Key | Value | Percentage Using at least Once
---------- | ----- | -----------
`#` | 2314 | 18.29%
`FASTA` | 6 | 3.66%
`Index-subfeatures` | 2 | 1.22%
`NOTE:` | 1 | 0.61%
`Type` | 1 | 0.61%
`attribute-ontology` | 1 | 0.61%
`color-code` | 1 | 0.61%
`date` | 2 | 1.22%
`feature-ontology` | 5 | 3.05%
`genome-build` | 1 | 0.61%
`gff-version	3` | 3 | 1.83%
`gff-version   3` | 1 | 0.61%
`gff-version 2` | 1 | 0.61%
`gff-version 3` | 1706 | 83.54%
`hdr` | 1 | 0.61%
`history` | 3 | 0.61%
`line-order` | 1 | 0.61%
`max-num-mismatches` | 1 | 0.61%
`max-read-length` | 1 | 0.61%
`primer-base` | 1 | 0.61%
`sequence-region` | 1720 | 56.10%
`solid-gff-version` | 1 | 0.61%
`source` | 1 | 0.61%
`source-ontology` | 1 | 0.61%
`source-version` | 1 | 0.61%
`species` | 6 | 3.66%
`time` | 1 | 0.61%

## Tabs

Tabs In Line | Count
---------- | -----
`9` | 207634
`8` | 2

## Tools

Tools | Count
---------- | -----
`feature` | 31
`GenBank` | 22
`.` | 18
`CPT_ShineFind` | 8
`CPT` | 8
`annotation` | 7
`RefSeq` | 6
`maker` | 6
`getOrfsOrCds` | 6
`blast` | 6
`blastn` | 6
`est2genome` | 6
`progressiveMauve` | 5
`AU9` | 5
`glimmer` | 4
`TMHMM` | 4
`example` | 3
`cpt.fixModel` | 3
`est` | 2
`aragorn` | 2

## Feature Types

Feature | Count
---------- | -----
`match_part` | 78299
`Z_over_Input` | 50198
`CDS` | 26841
`gene` | 10277
`match` | 7844
`Shine_Dalgarno_sequence` | 5813
`Match` | 5635
`mRNA` | 5555
`exon` | 4941
`contig` | 2204
`five_prime_UTR` | 2146
`three_prime_UTR` | 1733
`stop_codon` | 1425
`start_codon` | 1423
`Topological domain` | 898
`Transmembrane` | 454
`Chain` | 444
`terminator` | 260
`protein_match` | 126
`sequence_feature` | 106
`HSP` | 88
`intron` | 83
`sequence_variant` | 68
`non_canonical_five_prime_splice_site` | 47
`region` | 46
`transcript` | 45
`non_canonical_three_prime_splice_site` | 41
`tRNA` | 41
`mature_protein_region` | 37
`experimental_result_region` | 34
`long_terminal_repeat` | 32
`repeat_region` | 31
`remark` | 31
`binding_site` | 29
`ARS` | 26
`noncoding_exon` | 26
`polypeptide` | 25
`expressed_sequence_match` | 24
`Lipobox` | 20
`cds` | 18
`nucleotide_to_protein_match` | 16
`polypeptide_domain` | 15
`my_feature` | 10
`motif` | 10
`DNA` | 9
`misc_feature` | 8
`SNP` | 8
`transposable_element_gene` | 8
`supercontig` | 7
`regulatory` | 7
`micro_array_oligo` | 7
`nucleotide_match` | 7
`ncRNA` | 6
`promoter` | 6
`recombination_feature` | 6
`loop` | 6
`EST_match` | 6
`chromosome` | 5
`stop_codon_read_through` | 4
`pseudogene` | 4
`peptide_helix` | 4
`telomere` | 4
`LTR_retrotransposon` | 4
`mobile_genetic_element` | 3
`5'-UTR` | 3
`ncRNA_gene` | 3
`gene_component_region` | 3
`STS` | 3
`sequence_difference` | 3
`direct_repeat` | 3
`snoRNA` | 3
`BAC` | 2
`coding` | 2
`processed_transcript` | 2
`3'-UTR` | 2
`biological_region` | 2
`protein_coding_primary_transcript` | 2
`UTR` | 2
`centromere` | 2
`protein` | 1
`right_end_read` | 1
`left_end_read` | 1
`trace` | 1
`ultracontig` | 1
`protein_binding_site` | 1
`lincRNA_gene` | 1
`lincRNA` | 1
`signal_peptide` | 1
`origin_of_replication` | 1
`inverted_repeat` | 1
`scaffold` | 1
`clone_start` | 1
`clone_end` | 1
`snRNA` | 1

Feature (Using SO term) | Count
---------- | -----

## Scores?

Score Range | Count
---------- | -----
`Does Not Use Scores` | 103
`[0, 100]` | 28
`[0, 10000]` | 13
`[-2.8, 8728.0]` | 2
`[102.0, 17680.0]` | 2
`[-4438.107454, -183.539619]` | 1
`[-11.2071, 9.9974]` | 1
`[-37.0, 336.0]` | 1
`[-37.0, 207.0]` | 1
`[0, 1000]` | 1

## Tags

Tag        | Value | Percentage Using at least Once
---------- | ----- | ------------------------------
`Parent` | 125841 | 39697.48%
`qseq` | 69585 | 21951.10%
`sseq` | 69585 | 21951.10%
`ID` | 66745 | 21055.21%
`__.__` | 50198 | 15835.33%
`Name` | 35622 | 11237.22%
`Target` | 9143 | 2884.23%
`blast_gaps` | 8319 | 2624.29%
`blast_score` | 8319 | 2624.29%
`product` | 6136 | 1935.65%
`Target_protein` | 6014 | 1897.16%
`date_last_modified` | 5953 | 1877.92%
`date_creation` | 5953 | 1877.92%
`Gap` | 5901 | 1861.51%
`blast_bitscore` | 5504 | 1736.28%
`blast_evalue` | 5504 | 1736.28%
`blast_gapopen` | 5504 | 1736.28%
`blast_length` | 5504 | 1736.28%
`blast_mismatch` | 5504 | 1736.28%
`blast_nident` | 5504 | 1736.28%

## Percent Encoding

Tag        | Value | Count
---------- | ----- | -----
%20 | ` ` | 13404
%2A | `*` | 5069
%2C | `,` | 3123
%2B | `+` | 1415
%7C | `|` | 1327
%28 | `(` | 979
%29 | `)` | 979
%40 | `@` | 856
%3B | `;` | 832
%27 | `'` | 596
%25 | `%` | 154
%23 | `#` | 118
%C3 | `�` | 115
%C2 | `�` | 106
%2F | `/` | 74
%C4 | `�` | 26
%3D | `=` | 10
%87 | `�` | 8
%80 | `�` | 6
%83 | `�` | 6

## Non-percent encoded values

Tag        | Count
---------- | -----
` ` | 86193

## Trailing Semicolon in field 9

Tag        | Count
---------- | -----
`random-gff3s-from-helena/yeast_chr1+2.gff3` | 616
`random-gff3s-from-helena/Genus_species.gff3` | 116
`random-gff3s-from-helena/picard_CollectRnaSeqMetrics.gff3` | 69
`random-gff3s-from-helena/annot_mapped.gff3` | 38
`gffutils/glimmer_nokeyval.gff3` | 4
`gffutils/gms2_example.gff3` | 4
