# Statistics

## Headers

Header Key | Value | Percentage Using at least Once
---------- | ----- | -----------
`#` | 2313 | 20.71%
`FASTA` | 4 | 2.86%
`Index-subfeatures` | 2 | 1.43%
`NOTE:` | 1 | 0.71%
`date` | 1 | 0.71%
`feature-ontology` | 3 | 2.14%
`gff-version	3` | 3 | 2.14%
`gff-version   3` | 1 | 0.71%
`gff-version 3` | 1696 | 90.71%
`sequence-region` | 1684 | 62.14%
`source` | 1 | 0.71%
`species` | 5 | 3.57%

## Tabs

Tabs In Line | Count
---------- | -----
`9` | 207573
`8` | 2

## Tools

Tools | Count
---------- | -----
`progressiveMauve` | 75717
`.` | 57783
`AU9` | 20026
`blast` | 9939
`GenBank` | 8761
`feature` | 5947
`BLASTP` | 5629
`getOrfsOrCds` | 4240
`CPT` | 3800
`cpt.fixModel` | 3180
`CPT_ShineFind` | 2522
`SL2.40_assembly` | 2198
`TMHMM` | 1353
`SGD` | 1351
`RefSeq` | 799
`ena` | 792
`glimmer` | 660
`example` | 342
`Glimmer3` | 258
`maker` | 228
`protein_coding` | 191
`TAIR10` | 188
`AUGUSTUS` | 158
`GeneMark.hmm` | 142
`test_source` | 130
`funannotate` | 116
`blastn` | 114
`Genbank` | 109
`est2genome` | 108
`Araport11` | 105
`hg19_knownGene` | 69
`retained_intron` | 56
`repeatmasker` | 40
`snap` | 38
`est` | 34
`ALN` | 34
`aragorn` | 32
`augustus` | 27
`nonsense_mediated_decay` | 22
`exonerate:est2genome` | 22
`Gene3D` | 22
`processed_transcript` | 21
`Pfam` | 21
`CPT_LipoRy` | 20
`ITAG_eugene` | 19
`phytozome6` | 19
`SUPERFAMILY` | 17
`Phobius` | 17
`tbl2gff3` | 12
`PRINTS` | 11
`SMART` | 11
`landmark` | 11
`antisense` | 10
`ProDom` | 9
`exonerate:ungapped:dna2dna` | 8
`exonerate:protein2genome:local` | 8
`annotation` | 7
`InteinFinder` | 7
`Coils` | 7
`zonk` | 7
`CDD` | 6
`exonerate` | 6
`processed_pseudogene` | 4
`PANTHER` | 4
`predicted` | 4
`unprocessed_pseudogene` | 3
`PIRSF` | 3
`POPB` | 3
`snRNA` | 2
`rRNA` | 2
`ena_mobile_element` | 2
`exonerate:coding2coding` | 2
`ProSitePatterns` | 2
`POPA` | 2
`POPC` | 2
`ASM584v2` | 1
`TIGRFAM` | 1
`Contig` | 1
`bare_predicted` | 1

## Feature Types

Feature | Count
---------- | -----
`match_part` | 78299
`Z_over_Input` | 50198
`CDS` | 26826
`gene` | 10271
`match` | 7844
`Shine_Dalgarno_sequence` | 5813
`Match` | 5635
`mRNA` | 5551
`exon` | 4931
`contig` | 2204
`five_prime_UTR` | 2143
`three_prime_UTR` | 1729
`stop_codon` | 1425
`start_codon` | 1423
`Topological domain` | 898
`Transmembrane` | 454
`Chain` | 444
`terminator` | 260
`protein_match` | 126
`sequence_feature` | 89
`HSP` | 88
`intron` | 83
`sequence_variant` | 68
`non_canonical_five_prime_splice_site` | 47
`region` | 46
`transcript` | 45
`non_canonical_three_prime_splice_site` | 41
`tRNA` | 41
`mature_protein_region` | 37
`experimental_result_region` | 34
`long_terminal_repeat` | 32
`repeat_region` | 31
`remark` | 31
`binding_site` | 29
`ARS` | 26
`noncoding_exon` | 26
`polypeptide` | 25
`expressed_sequence_match` | 24
`Lipobox` | 20
`cds` | 18
`nucleotide_to_protein_match` | 16
`polypeptide_domain` | 15
`my_feature` | 10
`motif` | 10
`DNA` | 9
`SNP` | 8
`transposable_element_gene` | 8
`supercontig` | 7
`misc_feature` | 7
`regulatory` | 7
`micro_array_oligo` | 7
`nucleotide_match` | 7
`ncRNA` | 6
`promoter` | 6
`recombination_feature` | 6
`loop` | 6
`EST_match` | 6
`chromosome` | 5
`stop_codon_read_through` | 4
`pseudogene` | 4
`peptide_helix` | 4
`telomere` | 4
`LTR_retrotransposon` | 4
`mobile_genetic_element` | 3
`5'-UTR` | 3
`ncRNA_gene` | 3
`gene_component_region` | 3
`STS` | 3
`sequence_difference` | 3
`direct_repeat` | 3
`snoRNA` | 3
`BAC` | 2
`coding` | 2
`processed_transcript` | 2
`3'-UTR` | 2
`biological_region` | 2
`protein_coding_primary_transcript` | 2
`UTR` | 2
`centromere` | 2
`right_end_read` | 1
`left_end_read` | 1
`trace` | 1
`ultracontig` | 1
`protein_binding_site` | 1
`lincRNA_gene` | 1
`lincRNA` | 1
`signal_peptide` | 1
`origin_of_replication` | 1
`inverted_repeat` | 1
`scaffold` | 1
`clone_start` | 1
`clone_end` | 1
`snRNA` | 1
